### ISSUE TITLE
Compiling HPC-GAP with system ZeroMQ does not work

### ISSUE THREAD

**Issue by markuspf** [gap-system/gap#351]
When trying to build HPC-GAP using

```
./make.hpc ZMQ=/usr/local
```
I get an error 
```
NameError: name 'add_include_path' is not defined:
  File "/home/makx/ac/hpcgap/SConstruct", line 287:
    add_include_path(GAP["zmq"] + "/include")
*** Error code 2

Stop.
```

I know why it happens but didn't want to dig through the ordering of this file just yet, because my simple minded approach to fix it lead to ordering problems wrt `libgc`, i.e. it lead to the system `libgc` being used which does not seem to contain `GC_pthread_join` for some reason.

**Comment by rbehrends** [comment]
PR #352 is an attempt to address this issue. Can you verify if it works?

**Comment by alex-konovalov** [comment]
@markuspf, can we close this now, since it works?

**Comment by markuspf** [comment]
This is resolved by #352.
